fe/fe-core/src/main/java/com/starrocks/catalog/Index.java

```diff
@@ -47,16 +51,25 @@
 import java.io.DataOutput;
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Locale;
 import java.util.Map;
+import java.util.Map.Entry;
 import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 /**
  * Internal representation of index, including index type, name, columns and comments.
  * This class will used in olaptable
  */
 public class Index implements Writable {
+
+    @SerializedName(value = "indexId")
+    private long indexId;
```

Comment:
also this field.
Why do you introduce indexId? can you elaborate on it?

Reply:
We can use monotonically increasing id indexId based on `OlapTable` to uniquely identify an index, when we add a multi-column index we can name the index with the indexId, or we delete an old index then add a new one with the same indexName we can also distinguish from them